### DISCUSSION TITLE
blog/photopea-vs-photoshop

### DISCUSSION THREAD

**Discussion by giscus** [doceazedo/doce.sh#7] [Comments]
# blog/photopea-vs-photoshop

Esse editor de fotos online e gratuito é simplesmente superior ao Photoshop e eu vou te mostrar o porquê

https://pbe.doceazedo.com/blog/photopea-vs-photoshop

**Comment by Rozkovo** [comment]
Entregou tudo sem prometer nada real! 
Obrigado pelo artigo, Dooooce! <3
